### PR TITLE
Upgrade EDC version

### PR DESCRIPTION
**Pull request by algattik**
Upgrade EDC and webapp to latest version to maintain evolving compatibility.

Also rearranged some files for system tests for clarity.

### CODE REVIEW

**Review by cpeeyush — APPROVED** [review]

**Comment by cpeeyush** [comment]
@algattik very nit: Github issue link is missing.

File: launcher/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ EXPOSE 9191
 EXPOSE 8282
 
 # health status is determined by the availability of the /health endpoint
-HEALTHCHECK --interval=5s --timeout=5s --retries=10 CMD curl --fail http://localhost:8181/api/check/health
+HEALTHCHECK --interval=5s --timeout=5s --retries=10 CMD curl -H "X-Api-Key: $EDC_API_AUTH_KEY" --fail http://localhost:8181/api/check/health
```
Comment (marcgs) [inline launcher/Dockerfile:20]: Why is this needed now?

**Review by marcgs — APPROVED** [review]

**Review by ouphi — APPROVED** [review]

File: launcher/Dockerfile (hunk 1 above)
Comment (algattik) [reply launcher/Dockerfile:20]: change in control API
